src/main/java/org/eclipse/uprotocol/cloudevent/factory/UCloudEvent.java

```diff
@@ -121,16 +122,16 @@ static Optional<String> getToken(CloudEvent cloudEvent) {
     /**
      * Extract the integer value of the communication status attribute from a cloud event. The communication status attribute is optional.
      * If there was a platform communication error that occurred while delivering this cloudEvent, it will be indicated in this attribute.
-     * If the attribute does not exist, it is assumed that everything was Code.OK_VALUE.
+     * If the attribute does not exist, it is assumed that everything was UCode.OK_VALUE.
      * @param cloudEvent CloudEvent with the platformError to be extracted.
-     * @return Returns a {@link Code} value that indicates of a platform communication error while delivering this CloudEvent or Code.OK_VALUE.
+     * @return Returns a {@link Code} value that indicates of a platform communication error while delivering this CloudEvent or UCode.OK_VALUE.
      */
     static Integer getCommunicationStatus(CloudEvent cloudEvent) {
         try {
             return extractIntegerValueFromExtension("commstatus", cloudEvent)
-                    .orElse(Code.OK_VALUE);
+                    .orElse(UCode.OK_VALUE);
         } catch (Exception e) {
-            return Code.OK_VALUE;
+            return UCode.OK_VALUE;
```

Comment:
I would expect an exception to be anything but an OK_VALUE.
Is this intended or a mistake

Reply:
fixed thanks